generator/generator-opc-generator/generators/app/templates/src/styles/style.scss

```diff
@@ -0,0 +1,28 @@
+* {
+  font-size: 200%;
```

Comment:
Why these styles?
font-size: 200% (pls justify)

Reply:
It was for a the previous build. Removed the font-size